### PR TITLE
Better labeling

### PR DESCRIPTION
**Pull request by tarek-nawara**

### CODE REVIEW

File: scripts/labeled_stream.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+"""
```
Comment (mohamedramadan2515) [inline scripts/labeled_stream.py]: remove this script

File: scripts/labeled_stream_two.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+"""
```
Comment (mohamedramadan2515) [inline scripts/labeled_stream_two.py]: remove this script

File: scripts/sentiment.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import json
```
Comment (mohamedramadan2515) [inline scripts/sentiment.py]: remove this script

File: src/main/scala/edu/twitter/classification/Classifier.scala [hunk 4]
```diff
@@ -25,23 +28,31 @@ class Classifier(ssc: StreamingContext) {
     * @return stream of `ClassifiedTweets`
     */
   def createClassifiedStream(): DStream[ClassifiedTweet] = {
+    // @tarek-nawara Only checking for url patterns, mentions, hashtags and retweets
```
Comment (mohamedramadan2515) [inline src/main/scala/edu/twitter/classification/Classifier.scala]: remove @ tarek-nawara

File: src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala [hunk 5]
```diff
@@ -70,23 +62,11 @@ class SentimentModelDataCreator(tweetsRDD: RDD[Row]) {
     * sad by the presence of other words.
```
Comment (mohamedramadan2515) [inline src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala]: change the documentation

File: src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala [hunk 6]
```diff
@@ -18,20 +18,12 @@ class SentimentModelDataCreator(tweetsRDD: RDD[Row]) {
     * @return training data set, testing data set
     */
   def getTrainingAndTestingData(): (RDD[LabeledPoint], RDD[LabeledPoint]) = {
```
Comment (mohamedramadan2515) [inline src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala]: can you change the documentation of this method

File: scripts/labeled_stream.py (hunk 1 above)
Comment (tarek-nawara) [reply scripts/labeled_stream.py]: Can't remove it from this branch, will remove it from master

File: src/main/scala/edu/twitter/classification/Classifier.scala (hunk 4 above)
Comment (tarek-nawara) [reply src/main/scala/edu/twitter/classification/Classifier.scala]: Sure will do it.

File: src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala (hunk 5 above)
Comment (tarek-nawara) [reply src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala]: Sure will do it.

File: src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala (hunk 6 above)
Comment (tarek-nawara) [reply src/main/scala/edu/twitter/model/SentimentModelDataCreator.scala]: fixed :+1:

File: src/main/scala/edu/twitter/model/TweetsLoader.scala [hunk 7]
```diff
@@ -33,15 +30,14 @@ class TweetsLoader(sc: SparkContext) {
     * @return tweets that contain happy word , tweets contains sad word
     */
   private def cleanRecords(messages: DataFrame) : RDD[Row] = {
```
Comment (mohamedramadan2515) [inline src/main/scala/edu/twitter/model/TweetsLoader.scala]: can you change this function documentation

**Comment by mohamedramadan2515** [comment]
Do you remove scripts after merging?

**Pull request merged**